File: integration-tests-java/src/main/java/io/micronaut/jimmer/java/it/config/error/UserInfoErrorCode.java
package io.micronaut.jimmer.java.it.config.error;

import org.babyfish.jimmer.error.ErrorFamily;
import org.babyfish.jimmer.error.ErrorField;

@ErrorFamily
public enum UserInfoErrorCode {
    @ErrorField(name = "illegalChars", type = Character.class, list = true)
    ILLEGAL_USER_NAME,

    PASSWORD_TOO_SHORT,

    PASSWORDS_NOT_SAME
}

File: integration-tests-kotlin/src/main/kotlin/io/micronaut/jimmer/kotlin/it/config/error/UserInfoErrorCode.kt
package io.micronaut.jimmer.kotlin.it.config.error

import org.babyfish.jimmer.error.ErrorFamily
import org.babyfish.jimmer.error.ErrorField

@ErrorFamily
enum class UserInfoErrorCode {
    @ErrorField(name = "illegalChars", type = Character::class, list = true)
    ILLEGAL_USER_NAME,

    PASSWORD_TOO_SHORT,

    PASSWORDS_NOT_SAME,
}
